src/basic/linux/btrfs_tree.h

```diff
@@ -1094,14 +1090,14 @@ struct btrfs_file_extent_item {
 	__u8 encryption;
 	__le16 other_encoding; /* spare for later use */
 
-	/* are we inline data or a real extent? */
+	/* are we __inline__ data or a real extent? */
```

Comment:
I don't think the change to the comment is intended.

Reply:
This and below are all intentional. Such spurious replacement is caused by build script of the kernel.
Previously, we copied kernel headers from include/uapi/linux, but now they are from usr/include/linux.
The build script of the kernel simply replace `inline` with `__inline__`, even the term is in comment.
Please report that to the kernel team.